src/main/java/com/limechain/cli/Cli.java

```diff
@@ -73,16 +75,19 @@ private Options buildOptions() {
         Option dbPathOption = new Option(null, DB_PATH, true, "RocksDB path");
         Option dbClean = new Option("dbc", DB_RECREATE, false, "Clean the DB");
         Option nodeKey = new Option(null, NODE_KEY, true, "HEX for secret Ed25519 key");
+        Option nodeMode = new Option("mode", NODE_MODE, true, "Node mode (light/full)");
```

Comment:
It's important for users to default mode information(in our case is full node mode)

Reply:
Added additional information: `By default is full.`